lib/elixir/lib/process.ex

```diff
@@ -17,6 +17,59 @@ defmodule Process do
   `Registry`, `Supervisor` and `Task` for building their systems and
   resort to this module for gathering information, trapping exits, links
   and monitoring.
+
+  ## Aliases
+
+  Aliases are a feature introduced in Erlang/OTP 24. An alias is a way
+  to refer to a PID in order to send messages to it. The advantage of using
+  aliases is that they can be deactivated even if the aliased process is still
+  running. If you send a message to a deactivated alias, nothing will happen.
+  This makes request/response scenarios easier to implement.
+
+  You can use `alias/0` or `alias/1` to set an alias, and then you can send
+  messages to that alias like you do with PIDs using `send/2`. To deactivate
+  an alias, you can use `unalias/1`. If you send a message to a deactivated alias,
+  nothing will happen.
+
+  For example, you could have a process that listens for `:ping` messages:
+
+      def server do
+        receive do
+          {:ping, source_alias} ->
+            send(source_alias, :pong)
+            server()
+        end
+      end
+
+  Now, another process might ping this process:
+
+      server = spawn(&server/0)
+
+      source_alias = Process.alias()
+      send(server, {:ping, source_alias})
+
+      receive do
+        :pong -> :pong
+      end
+      #=> :pong
+
+  If now you deactivate the `source_alias` and ping the server again, you
+  won't receive any response since the server will `send/2` the `:pong` response
+  to a deactivated alias.
+
+      Process.unalias(source_alias)
+      send(server, {:ping, source_alias})
+
+      receive do
+        :pong -> :pong
+      after
+        1000 -> :timeout
+      end
+      #=> :timeout
+
+  See also the [*Process Aliases*
```

Comment:
```suggestion
  See also the [Process Aliases
```

Reply:
This one is different since it's referring to the section name, not the term. Like, the section name is in italics because it's a section name 😄

Reply:
We don't use this notation elsewhere though. :( So we should either discuss if we want to do it consistently. Easier to postpone for now.

Reply:
Okay, sounds good!